pylsp/plugins/flake8_lint.py

```diff
@@ -81,13 +81,14 @@ def run_flake8(flake8_executable, args, document):
     try:
         cmd = [flake8_executable]
         cmd.extend(args)
-        p = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+        p = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE)  # pylint: disable=consider-using-with
     except IOError:
         log.debug("Can't execute %s. Trying with '%s -m flake8'", flake8_executable, sys.executable)
         cmd = [sys.executable, '-m', 'flake8']
         cmd.extend(args)
         p = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE)  # pylint: disable=consider-using-with
-    (stdout, stderr) = p.communicate(document.source.encode())
+    with p as p:
+        (stdout, stderr) = p.communicate(document.source.encode())
```

Comment:
I think this is what's breaking this plugin. Please use a `# pylint` comment to disable the warning instead.